content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md

```diff
@@ -116,3 +116,32 @@ Here are some helpful resources to get started with `policy-controller`:
 
 - [Installation](https://github.com/sigstore/helm-charts/tree/main/charts/policy-controller)
 - [Configuration Options](https://github.com/sigstore/policy-controller/tree/main/config)
+
+## Verify the Software Bill Of Materials (SBOM)
+
+The Kubernetes SBOM is signed as well and can be verified by using the sigstore
+certificate and signature, or the corresponding SHA files:
```

Comment:
```suggestion
## Verify SBOM

The Software Bill Of Materials (SBOM) for Kubernetes can be verified by using either the sigstore
certificate and signature or the corresponding SHA files:
```
Is it better to keep the heading concise and use a full spelling in the body text?

Reply:
It makes sense to spell out SBOM in the heading. In the sentence, you can spell out SBOM and provide the acronym.

```
## Verify the Software Bill of Materials

The Kubernetes Software Bill of Materials (SBOM) can be verified 

or
You can verify the Kubernetes Software Bill of Materials (SBOM) by using the sigstore certificate ...
```

Reply:
Changed as suggested, thanks folks! :+1: